docs/monitoring/10-01-prometheus-istio-server-oom.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Prometheus Istio Server restarting or in crashback loop due to OOM
```

Comment:
```suggestion
title: Prometheus Istio Server restarting or in crashback loop
```
The title looks a bit long on the website (taking 3 lines) and I think it's enough to mention the memory issue in the description.